data_storage/export_format.md

```diff
@@ -1,22 +1,28 @@
 Export Format
 -------------
 
-- **5.1.1** ![](/badge/req.png) A Toxcore data file saved in the root of the client's
-  working directory, with the file name `<profile_name>.tox`
+- **5.1.1** ![](/badge/req.png) The client will export files relative to a folder called
+  the export folder, which must be in a location accessible to the user.
 
-- **5.1.2** ![](/badge/req.png) If the client supports user avatars, they will be stored
-  under the `avatars/` directory in the root of the client's working directory.
+- **5.1.2** ![](/badge/rec.png) The user should be able to choose the location
+  of the export folder.
 
-- **5.1.3** ![](/badge/req.png) Avatars are named according to the following format:
+- **5.1.3** ![](/badge/req.png) A Toxcore data file saved in the root of the
+  export folder, with the file name `<profile_name>.tox`.
```

Comment:
@iphydf 
I strongly disagree. 
For portability, clients should only use the common toxcore API when exporting and importing *toxcore savedata*. 

Trying to standardize a completely different format when this is *very clearly* toxcore's job is either an indication of serious problems with toxcore, or severely misguided. 
If there are concerns with backwards compatibility or lacking documentation, this is entirely a QoI problem that toxcore needs to address.

Ignoring that I'm not convinced all the toxsave data is available through the public API, who would maintain your format and its specification as toxcore evolves? How is the extra maintenance burden in any way better than just fixing `tox_get_savedata`?
I will not parse third party homegrown formats for something the core library ought to provide.